src/governance_engine/gsri_engine.py

```
@@ -0,0 +1,45 @@
+        }
+        self.mitigations_active = False
+
+    def set_mitigations(self, active: bool):
+        self.mitigations_active = active
+
+    def calculate_score(self):
+        """
+        Calculates the current G-SRI score.
+        Score = Sum(Vector * Random_Factor) / (1 + Mitigation_Dampener)
+        """
+        base_score = sum(val * random.uniform(0.8, 1.2) for val in self.risk_vectors.values())
```

Ruff (0.15.17)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)